Shuttlerock/actions
# [STUDIO-1237] Add an owner and labels when creating a release PR

## Add an owner and labels when creating a release PR

[Jira Tech task](https://shuttlerock.atlassian.net/browse/STUDIO-1237)
[Jira Epic](https://shuttlerock.atlassian.net/browse/STUDIO-231)



## How to test the PR

Create a release PR, and check that it has both an assigned owner and some labels.

## Deployment Notes

 None
